pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -82,20 +94,20 @@ func LoadClientCert(kubeconfigPath string, bootstrapPath string, certDir string,
 	if err != nil {
 		return fmt.Errorf("unable to build bootstrap client cert path: %v", err)
 	}
+	defer func() {
```

Comment:
why is this moved above RequestNodeCertificate as well? Does that write anything?

Reply:
To avoid leaving a cert on disk if we got a key but failed during req node certificate.  The cert is useless without the key (which we delete above).  This makes the flow a lot more predictable - if we fail to attain a cert, delete key and cert and try again.